TicTacToe2/TicTacToeGame.cs

```diff
@@ -90,73 +89,37 @@ private void DrawBoard()
 
         private void GetUserInput()
         {
-            string choiceText;
             bool validInput = false;
-            bool correctNumber = false;
 
             while (!validInput)
             {
                 Console.WriteLine($"\nIt is player {player}'s turn.");
                 Console.Write("Pick a number from the board to place your mark: ");
-                choiceText = Console.ReadLine();
-
-                bool isParsable = Int32.TryParse(choiceText, out choice);
 
-                if (!isParsable)
+                if (gridNumber == 1)
                 {
-                    Console.WriteLine("\nPlease enter a number.");
-                    validInput = false;
+                    choice = GetUserInput(1, 9);
```

Comment:
The choice variable is now only used by this function and should be removed from the class